src/linuxX64Main/kotlin/mu/internal/KLogger.kt

```diff
@@ -0,0 +1,76 @@
+package mu.internal
+
+import mu.KLogger
+import mu.KotlinLoggingConfiguration.APPENDER
+import mu.Marker
+
+internal class NativeLogger(
+    private val loggerName: String
+) : KLogger {
+
+    override fun entry(vararg argArray: Any?) {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
same for TODO's in this file

Reply:
I'll address those. First I want to clarify whether we should switch to platform snprintf or use the lib.